Code/Tools/AssetProcessor/native/tests/UnitTestUtilities.h

```diff
@@ -198,7 +199,7 @@ namespace UnitTests
                         // Any mode besides OPEN_READ_ONLY creates a file
                         if ((systemMode & ~int(IO::SystemFile::SF_OPEN_READ_ONLY)) > 0)
                         {
-                            m_mockFiles[handle] = "";
+                            m_mockFiles[handle] = AZStd::pair<AZStd::string, AZStd::string>(filePath, "");
```

Comment:
you may also be able to get away with this syntax or similar
```suggestion
                            m_mockFiles[handle] = {filePath, ""};
```

Reply:
That did indeed work :)